aquarius/ddo_checker/schemas/metadata_local_v0_6.json

```diff
@@ -443,6 +443,55 @@
           "pattern": "^(.*)$"
         }
       }
+    },
+    "credentials": {
+      "default": null,
+       "properties": {
+          "allow": {
+            "$id": "#/properties/credentials/properties/allow",
+            "type": "array",
+            "title": "The Credentials Schema",
```

Comment:
The rest of this file does a pretty good job of giving each Schema a unique title.  I recommend de-conflicting "The Credentials Schema"s by making them a little more specific like

* "The Credentials allow Schema"
* "The Credentials deny Schema"